content/rdi/architecture.md

```diff
@@ -97,3 +127,28 @@ Debezium Server should run in warm High Availability topology in one of the foll
 - Running on 2 VMs in an active/passive setup. The failover orchestration can be done by tools such as [Pacemaker](https://clusterlabs.org/pacemaker/doc/).
 - Running as a Kubernetes pod where Kubernetes watches the pod liveliness and readiness and acts to recover nonfunctional pod
   It is important to note that Redis Data Integration does not keep any state on Debezium. All state is kept in Redis Data Integration.
+
+## Write behind functionality and architecture
+
+RDI write behind allows integration between Redis Enterprise (as the source of changes to data) to downstream databases or datastores.
+RDI captures the changes to a selected set of key-patterns in a Redis keyspace and asynchronously writes them in small batches to the downstream database, so the application doesn't need to take care of remodeling the data or managing the connection with the downstream database.
+
+RDI write behind can normalize a key in Redis to several records in one or more tables at the target.
+To learn more about write behind declarative jobs and renormalization read the [write behind quick start guide]({{<relref="/rdi/quickstart/write-behind-guide.md">}})
```

Comment:
One more fix... the relref shortcode syntax was incorrect (no = sign).

```suggestion
To learn more about write behind declarative jobs and renormalization read the [write behind quick start guide]({{< relref "/rdi/quickstart/write-behind-guide.md" >}})
```

Reply:
done